pytorch_transformers/tokenization_utils.py

```diff
@@ -503,6 +503,9 @@ def add_tokens(self, new_tokens):
         if not new_tokens:
             return 0
 
+        if len(new_tokens) != len(set(new_tokens)):
```

Comment:
Maybe this could just be another condition on line 512 instead of an error raised?
like
```
if token != self.unk_token and \
    self.convert_tokens_to_ids(token) == self.convert_tokens_to_ids(self.unk_token) and \
    token not in to_add_tokens:
```

Reply:
Ah, good point, thanks! Updated ✅